docs/advanced/mbed_targets.md

```diff
@@ -1,6 +1,6 @@
 # Adding and configuring mbed targets
 
-mbed uses JSON as a description language for its build targets. The JSON description of mbed targets can be found in `targets/targets.json`. To better understand how a target is defined, we'll use this example (taken from `targets.json`):
+mbed uses JSON as a description language for its build targets. The JSON description of mbed targets can be found in `targets/targets.json` and in `custom_targets.json` in the root of a project directory. Definitions of new targets in a `custom_targets.json` will be added to the list of available targets. Any re-definitions of existing targets will be merged with the original definition by replacing values found in `targets/targets.json` with values found in the same key in `custom_targets.json`. To better understand how a target is defined, we'll use this example (taken from `targets.json`):
```

Comment:
Nice clarification. :)

I recommend changing the following:

- Change "The JSON description of mbed targets can be found in ..." to "You can find the JSON description of mbed targets in ..." for active voice.
- Change the sentence starting with "Definitions of new targets" to active voice. Who or what is adding the definitions to the list of available targets?
- Delete hyphen in "redefinitions".
- Change sentence starting with "Any redefinitions of existing targets" to active voice. Who or what is merging the redefinitions with original definitions?